ntoskrnl/cc/view.c

```diff
@@ -411,8 +412,7 @@ CcRosFlushDirtyPages (
         if (--SharedCacheMap->OpenCount == 0)
             CcRosDeleteFileCache(SharedCacheMap->FileObject, SharedCacheMap, &OldIrql);
 
-        if (!NT_SUCCESS(Status) && (Status != STATUS_END_OF_FILE) &&
-            (Status != STATUS_MEDIA_WRITE_PROTECTED))
+        if (!NT_SUCCESS(Status))
```

Comment:
I see what you're fixing here, but it doesn't seem to relate to the call in `CcFlushCache`. Are you saying Timo fixed the wrong instance?

Reply:
> Are you saying

No, the 3 commits are independent.
I concur with what you wrote on his PR and I have no additional clue about that.
I simply noticed this separate case while reviewing the other one.